### PR TITLE
Added 429 as possible response status code

### PR DESCRIPTION
**Pull request by mcardosos**
PTAL
@jhendrixMSFT @marstr 
This will allow us to gen from `web` swagger

### CODE REVIEW

File: src/CodeNamerGo.cs [hunk 1]
```diff
@@ -151,7 +151,8 @@ public CodeNamerGo()
                 HttpStatusCode.BadGateway,
                 HttpStatusCode.ServiceUnavailable,
                 HttpStatusCode.GatewayTimeout,
-                HttpStatusCode.HttpVersionNotSupported
+                HttpStatusCode.HttpVersionNotSupported,
+                (HttpStatusCode)429
```
Comment (marstr) [inline src/CodeNamerGo.cs]: Nit it seems like these are clustered by their first digit. That would mean that 429 is currently lumped with the 500s, but it should be with the 400s.

Separately, I've verified that this doesn't seem to be part of .NET's Status enumeration, documentation can be found here: https://msdn.microsoft.com/en-us/library/system.net.httpstatuscode(v=vs.110).aspx

Comment (mcardosos) [reply src/CodeNamerGo.cs]: Fixed, lololol

Comment (marstr) [reply src/CodeNamerGo.cs]: Lol race conditions

File: src/CodeNamerGo.cs [hunk 2]
```diff
@@ -145,6 +145,7 @@ public CodeNamerGo()
                 HttpStatusCode.UnsupportedMediaType,
                 HttpStatusCode.RequestedRangeNotSatisfiable,
                 HttpStatusCode.ExpectationFailed,
+                (HttpStatusCode)429,
```
Comment (jhendrixMSFT) [inline src/CodeNamerGo.cs]: Can you define this as a const?

**Comment by mcardosos** [comment]
Merging when Johannes goes green

**Pull request merged**